charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -35,6 +35,9 @@ spec:
 {{- toYaml .Values.controller.podLabels | nindent 8 }}
         {{- end }}
     spec:
+      {{- if .Values.global.securityContext.enabled }}
```

Comment:
Switch this to `.Values.global.securityContext` to pair with requested Values change below.